build.sbt

```diff
@@ -112,7 +114,7 @@ lazy val rdfTestSuite = crossProject
   .settings(commonSettings: _*)
   .settings(
     name := "banana-test",
-    libraryDependencies += scalatest,
+    libraryDependencies += "org.scalatest" %%% "scalatest" % scalatestVersion,
```

Comment:
why this change? The scalatest val is still available in `project/Dependencies.scala` as far as I can see reading the web view here.

Reply:
It is because in Dependencies.scala there is a statement:
> Note: %%% can only be used within a task or setting macro, such as :=, +=, ++=, Def.task, or Def.setting...

Yesterday I read at the scala-js gitter channel:
> Ah, in a `.scala` file you need an additional import: `import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._`

I already tested but it is still not working stating:

`Dependencies.scala:33:47: value %%% is not a member of String`

This is one of the root causes, that tests aren't running and scala-js dependecies are not set correctly (which is especially bad if a library wants to use `banana-rdf-sjs` without manually adding `scalaz`)

Reply:
It seems to be used here
https://github.com/portable-scala/sbt-crossproject/blob/master/sbt-crossproject/src/main/scala/sbtcrossproject/CrossPlugin.scala
and here
https://github.com/scala-js/scala-js/blob/master/sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

Reply:
The `%%%` method is defined there, with `private[sbtplatformdeps]` in its parent class signature.

https://github.com/portable-scala/sbt-platform-deps/blob/master/src/main/scala/org/portablescala/sbtplatformdeps/PlatformDepsBuilders.scala

Until we know how to declare it in `project/Dependecies.scala`, IMHO we should take the imperfect solution, as the current release of banana-rdf can be considered broken -  sjs-libs are neither tested nor are they published with correct dependencies.

Reply:
ok, just open an issue and link to it, in the dependencies with the explanation. Then it will be easier to remember to do something when the right time comes.